lib/Conversion/TorchToArith/TorchToArith.cpp

```diff
@@ -189,8 +189,7 @@ class ConvertTorchTensorLiteralOp
               RankedTensorType::get(type.getShape(), builtinTensorElemTy);
           AsmResourceBlob *blob = elements.getRawHandle().getBlob();
           assert(blob && "Expecting dense resource with a valid blob");
-          rewriter.replaceOpWithNewOp<arith::ConstantOp>(
-              op, DenseElementsAttr::get(shapedType, blob->getData()));
+          rewriter.replaceOpWithNewOp<arith::ConstantOp>(op, DenseResourceElementsAttr::get(shapedType, elements.getRawHandle()));
```

Comment:
What's the motivation for this change?

Reply:
Hi, Ramiro 
We want to roll Turbine and start using DenseResourceElementsAttr in place of DenseElementsAttr, which should give us a significant boost in terms of compiler memory usage. 

https://github.com/openxla/iree/pull/14980

Reply:
Thanks for the context!